include/vkgcDefs.h

```diff
@@ -70,6 +70,7 @@
 //* %Version History
 //* | %Version | Change Description                                                                                    |
 //* | -------- | ----------------------------------------------------------------------------------------------------- |
+//* |     46.0 | Add pipelineCacheAccess, stageCacheAccess(es) to GraphicsPipelineBuildOut/ComputePipelineBuildOut     |
```

Comment:
Since you just add new members to structure info, I think a minor version bump is enough 45.1. If we remove some existing or change the interface method, we need to do a major version bump.